Automattic/wp-calypso
# Update the license assignment page

Update the license assignment page query param for multi license from "keys" to "products" Also update all instances of singular license to plural when the products query param is used and there is more than one product license being issued

#### Proposed Changes

* Update the expected query param in the licenses assignment page from `keys` to `products`
* Update the page title
* Update other instances from singular license to licenses including 
* 
<img width="1421" alt="Screenshot 2022-11-08 at 2 53 08 PM" src="https://user-images.githubusercontent.com/1273880/200684127-2b85aed7-966e-4bc4-9170-7b3ca5f2a96e.png">


#### Testing Instructions



* Apply this PR.
* Run yarn-start-jetpack-cloud.
* Test the licenses assignment page with the singular `key` query param by attempting to assign any license from `http://jetpack.cloud.localhost:3000/partner-portal/licenses`.
- * Ensure that the singular "license" is used.
* Update the query params to include `products`.
>* Ensure a single product returns the expected singular license on the page test.
>* Ensure that a comma separated listed on the `products` query param updates the page text.

#### Pre-merge Checklist






Related to #
